docs/docs/aztec/concepts/accounts/index.md

```diff
@@ -2,58 +2,49 @@
 title: Accounts
 sidebar_position: 1
 tags: [accounts]
----
-
-**Every account in Aztec is a smart contract** which defines the rules for whether a transaction is or is not valid. This allows implementing different schemes for transaction signing, nonce management, and fee payments. However, encryption and nullifying keys, which are specific to private blockchains, are still enshrined at the protocol level.
-
-In this section, you’ll learn about how Aztec defines AA (account abstraction) and its correlation with encryption keys and nullifying keys. We’ll go through:
+---                                                                                                                                                               
 
-- The importance and implications of AA
-- Understanding account contracts and wallets in relation to Aztec
-- Concept of authorization and actions along with encryption
-- The future of fee management in Aztec
+Aztec has native account abstraction. Every account in Aztec is a smart contract.
 
-## Background
+In this section, you’ll learn about Aztec’s account abstraction, Aztec accounts and address derivation, how wallets relate to accounts, and how the entrypoints are defined.
```

Comment:
Should we use `’` or `'` across the board? And yes, this is a major cause for concern, right after tabs vs spaces.